index.html

```diff
@@ -4,36 +4,57 @@
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Portafolio</title>
 </head>
-
 <body>
     <header>
      <section class="home">
     <div class="cover">
-        <div class="bar-status">
-            <time> 9:41</time>
-            <img src="statusphone.jpg" alt="statusphone">
+        <div class="status">
+        <div class="status-mobile">
+            <time datetime="2022-08-22"> 9:41</time>
+        <div>
+        <!--phone signal-->
+        <i class="bi bi-reception-4"></i>
+        <!--wifilogo-->
+        <i class="bi bi-wifi"></i>
+        <!--battery file-->
+        <i class="bi bi-battery-full"></i>
         </div>
-        <div class="main">
-            <p>cesar valencia</p>
-            <button>hamburger</button>
+    </div>
+        <div class="nav-bar">
+            <p class="name">Cesar Valencia</p>
+          <div>
+        <!--menu-->
+            <i class="bi bi-list menu"></i>
+          </div>
         </div>
+    </div>
         <div class="content">
-            <h1>Hey there, i´m cesar</h1>
-            <h2> im a software engineer and mecatronic ensgineer</h2>
-            <p><i>can help you build a product , 
-                feature or website Look through some of my work and experience! 
-                If you like what you see and have a project you need coded, 
-                don’t hestiate to contact me.</i></p>
-        </div>
-        <div>
-            <input type="image"  alt="githublink" src="imgs/github image.jpg"/>
-            <input type="image"  alt="linkeinlink" src="imgs/inlinked image.jpg"/>
-            <input type="image"  alt="twitterlink" src="imgs/twitter img.jpg"/>
+            <h1>Hey there,<br>I´m cesar</h1>
+            <h2> I'am a software engineer and mecatronic engineer</h2>
+            <p><em>can help you build a product,feature or website Look
+                through some of my work and experience!If you like what you
+                see and have a project you need coded, don’t hestiate to
+                contact me.</em></p>
         </div>
-    </div> 
+    <h3> Let´s Connect</h3>
+<div>
+        <nav>
+          <input type="image" src="imgs/githubimg.png" alt="githubimage">
+          <input type="image" src="imgs/linkeinimg.png" alt="linkedinimage">
+          <input type="image" src="imgs/twitter.png" alt="twitterimg">
+        </nav>
```

Comment:
- Kindly, by convention the "nav" tag is the one that contains the main menu of your site, you should not use it in this section.
- Kindly, the images are usually called with the "IMG" tag, it would be good if you get used to using it, it will help you a lot in the future.
- Kindly, image listings should be UL/LI listed.

Reply:
ok i change it and use a div to put these images in the screen